test/vmtests/vmtests/utils/libvirt_utils.py

```diff
@@ -136,21 +155,57 @@ def _add_disk_xml(
     disk_source = ET.SubElement(disk, "source")
     disk_source.attrib["file"] = file_path
 
+# Adds a disk to a libvirt domain XML document.
+def _add_iso_xml(
+    devices: ET.Element,
+    file_path: str,
+    bus_type: str,
+    next_disk_indexes: Dict[str, int],
+) -> None:
+    device_name = "hda"
+
+    disk = ET.SubElement(devices, "disk")
+    disk.attrib["type"] = "file"
+    disk.attrib["device"] = "cdrom"
+
+    disk_driver = ET.SubElement(disk, "driver")
+    disk_driver.attrib["name"] = "qemu"
+    disk_driver.attrib["type"] = "raw"
+
+    disk_target = ET.SubElement(disk, "target")
+    disk_target.attrib["dev"] = device_name
+    disk_target.attrib["bus"] = bus_type
+
+    disk_source = ET.SubElement(disk, "source")
+    disk_source.attrib["file"] = file_path
+    disk_source.attrib["index"] = "1"
+
+    disk_readonly = ET.SubElement(disk, "readonly")
+
+    disk_alias = ET.SubElement(disk, "alias")
+    disk_alias.attrib["name"] = "ide0-0-0"
+
+    disk_address = ET.SubElement(disk, "address")
+    disk_address.attrib["type"] = "drive"
+    disk_address.attrib["controller"] = "0"
+    disk_address.attrib["bus"] = "0"
+    disk_address.attrib["target"] = "0"
+    disk_address.attrib["unit"] = "0"
 
 def _gen_disk_device_name(prefix: str, next_disk_indexes: Dict[str, int]) -> str:
     disk_index = next_disk_indexes.get(prefix, 0)
     next_disk_indexes[prefix] = disk_index + 1
 
-    match prefix:
-        case "vd" | "sd":
-            # The disk device name is required to follow the standard Linux device naming
-            # scheme. That is: [ sda, sdb, ..., sdz, sdaa, sdab, ... ]. However, it is
-            # unlikely that someone will ever need more than 26 disks. So, keep it simple
-            # for now.
-            if disk_index < 0 or disk_index > 25:
-                raise Exception(f"Unsupported disk index: {disk_index}.")
-            suffix = chr(ord("a") + disk_index)
-            return f"{prefix}{suffix}"
-
-        case _:
-            return f"{prefix}{disk_index}"
+    # cannot use python's 'match' because Azure Linux 2.0 has an older version
```

Comment:
It might be good to explicitly specify the minimum Python version we are targeting.

Reply:
You mean in a comment? or a runtime check?

Reply:
As a comment and/or in the README would be a good start.